docs/source/user_guide/components.ipynb

```diff
@@ -481,60 +455,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# custom component\n",
-    "from evalml.pipelines.components import Transformer\n",
-    "import pandas as pd\n",
+    "# We can also do this for custom components\n",
     "from evalml.pipelines.components.utils import generate_component_code\n",
     "\n",
-    "class MyDropNullColumns(Transformer):\n",
```

Comment:
No need to repeat this, I believe the only difference is the name and there's nothing special about this necessary for code gen so deleting!